src/python/WMCore/Services/WMStats/WMStatsReader.py

```diff
@@ -332,6 +341,47 @@ def getRequestByStatus(self, statusList, jobInfoFlag=False, limit=None, skip=Non
 
         return results
 
+    def getRequestByStatusAndStartTime(self, statusList, jobInfoFlag=False, detail=False, startTime=0):
```

Comment:
From the comment above, I think startTime should not default to 0 to avoid heavy calls in the system. It's probably better to make it a mandatory arg.